templates/pkg/resource/manager.go.tpl

```diff
@@ -16,19 +16,27 @@ import (
 	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
 	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
 	acktypes "github.com/aws-controllers-k8s/runtime/pkg/types"
+	acksvcv1alpha1 "github.com/aws-controllers-k8s/{{ .ServicePackageName }}-controller/apis/v1alpha1"
```

Comment:
@vijtrip2 what would you say to putting all of the resource reference generated code into a new file called `pkg/resource/{resource}/references.go`?

Reply:
Yes, I will incorporate it and send another revision.